Dockerfile

```
@@ -0,0 +1,23 @@
+FROM node:14
+
+ENV ENVIRONMENT=development
+
+WORKDIR /usr/src/app
+
+COPY package.json yarn.lock ./
+
+RUN yarn install
+
+COPY . .
+
+# Rename Firebase development configs based on OS
+RUN if [ "$(uname)" == "Darwin" ] || [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then \
+      yarn rename-dev-configs; \
+    else \
+      cp /ios/firebase-config.development.js /ios/firebase-config.js && \
```

The Firebase config file on iOS is called `GoogleService-Info.plist`

```suggestion
      cp /ios/GoogleService-Info.development.plist /ios/GoogleService-Info.plist && \
```